pkg/k8s/factory_functions.go

```diff
@@ -593,6 +593,7 @@ func ConvertToPod(obj interface{}) interface{} {
 			SpecHostNetwork:        concreteObj.Spec.HostNetwork,
 			SpecContainers:         containers,
 			StatusPhase:            concreteObj.Status.Phase,
+			StatusStartTime:        concreteObj.Status.StartTime,
```

Comment:
You also need to add this further below in the `cache.DeletedFinalStateUnknown` case as well

Reply:
I'm following alternative paths of investigation for the core issue at the moment but if I come back to this approach I'll address this.